samples/onnx_accelerator_sample1/onnxacceleratorsampleone/with_ggv2/components/aws.samples.windturbine.detector/recipe.yaml

```diff
@@ -0,0 +1,44 @@
+RecipeFormatVersion: "2020-01-25"
+ComponentName: "{COMPONENT_NAME}"
+ComponentVersion: "{COMPONENT_VERSION}"
+ComponentDescription: Detects anomalies for wind turbines
+ComponentPublisher: "{COMPONENT_AUTHOR}"
+ComponentConfiguration:
+  DefaultConfiguration:
+    model_name: "__MODEL_NAME__"
```

Comment:
`model_name` and `model_version` should be derived from the model component. Ideally, the model component contains both the onnx model and a metadata file with all the information relative to this model, possibly coming from the SM model registry